docs/resources/restriction_policy.md

```diff
@@ -0,0 +1,56 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_restriction_policy Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog RestrictionPolicy resource. This can be used to create and manage Datadog restriction_policy.
+---
+
+# datadog_restriction_policy (Resource)
+
+Provides a Datadog RestrictionPolicy resource. This can be used to create and manage Datadog restriction_policy.
+
+## Example Usage
+
+```terraform
+# Create new restriction_policy resource
+
+
+resource "datadog_restriction_policy" "foo" {
+  bindings {
+    principals = ["role:00000000-0000-1111-0000-000000000000"]
+    relation   = "editor"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `resource_id` (String) Identifier for the resource, formatted as type:id.
+
+### Optional
+
+- `bindings` (Block List) (see [below for nested schema](#nestedblock--bindings))
```

Comment:
what additional details would you want here? will test that out but should be consistent with our API because the principals and relation are marked as optional in the resource go file

Reply:
actually nvm, i saw that you added the details in [this](https://github.com/DataDog/terraform-provider-datadog/blob/79824a62707ac276117d297bed68a09027500546/docs/resources/restriction_policy.md#nested-schema-for-bindings) section